packages/tldraw/src/lib/ui/components/MainMenu/DefaultMainMenuContent.tsx

```diff
@@ -37,15 +36,54 @@ import { TldrawUiMenuSubmenu } from '../primitives/menus/TldrawUiMenuSubmenu'
 export function DefaultMainMenuContent() {
 	return (
 		<>
+			<FileSubmenu />
```

Comment:
I still don't think thing belongs as a default in the library. I don't think wanting this extra place for export justifies a menu that in a lot of contexts where people are using tldraw is just going to feel a bit weird and out of place. Really, unless you're using tldraw as a full-screen drop-in whiteboard, "File" isn't going to make sense. If you _are_, you probably want to override this and do your own thing anyway.

If we thought it was important to add separate options for exporting the entire tldraw document - maybe all pages as PNGs or something - then I could maybe see file being more justified there. As it is though, we're just doing the same "export the current selection" option which is already accessible in the edit & context menus.

Reply:
alright, sounds good — I see your position. I'll pull this part out and we can revisit later maybe 👍